projects/client-api/src/index.js

```diff
@@ -26,6 +26,7 @@ export class Client extends ClientBase {
      * @constructor
      * @param {string} username - Graphistry server username
      * @param {string} password - Graphistry server password
+     * @param {string} org - Graphistry organization (optional)
```

Comment:
We can use brackets [] too to indicate that org is optional:
`* @param {string} [org] - Graphistry organization (optional)`